cmp/tab/TabModel.ts

```diff
@@ -123,16 +138,19 @@ export class TabModel extends HoistModel {
             showRemoveAction && XH.isMobileApp,
             'Removable Tabs not supported in Mobile toolkit.'
         );
+        throwIf(childTabs && content, "A tab cannot define both 'content' and 'childTabs'. ");
```

Comment:
If we combined `childTabs` and `content`, could remove this check.

Reply:
👍🏻

Reply:
done